blocks/dashboard/renderUsersTab.js

```diff
@@ -440,8 +441,17 @@ export default async function renderUserTab({ container }) {
 
   let anonymousUserResponse = null;
   let anonymousUserData = null;
+  const anonymousFilter = {
+    event: readQueryParams().anonymousEvent || 'all',
+    location: readQueryParams().anonymousLocation || 'all',
+    referrer: readQueryParams().anonymousReferrer || 'all',
+    browser: readQueryParams().anonymousBrowser || 'all',
+    device: readQueryParams().anonymousDevice || 'all',
```

Comment:
If possible call `readQueryParams()` once, and use the properties from the returned Object. Each call generates `new URLSearchParams` and generates an object out of it. No point in running that over & over if we don't have to